### ISSUE TITLE
unable to build, download batik-1.13 not available

### ISSUE THREAD

**Issue by FalcoGer** [benfry/processing4#179]
## Description
When trying to call ant run for the first time it tries to download batik-bin-1.13.zip from https://apache.osuosl.org/xmlgraphics/batik/binaries/
However this fails since the file is not available

## Expected Behavior
A file is downloaded.

## Current Behavior
Build aborts because 404

## Steps to Reproduce
1. Clone repo
2. Enter build directory
3. ant run

## Your Environment
* Processing version: b5baaa409dcd5a34293e40de7b6d63368966cda1
* Operating System and OS version: Ubuntu Mate 20.04
* Other information:
openjdk 11.0.10 2021-01-19
OpenJDK Runtime Environment (build 11.0.10+9-Ubuntu-0ubuntu1.20.04)
OpenJDK 64-Bit Server VM (build 11.0.10+9-Ubuntu-0ubuntu1.20.04, mixed mode, sharing)

## Possible Causes / Solutions
Upload the required file to a server and download from there.
Alternatively make this batik guy not delete old versions
Alternatively use libraries that are available through software repositories and prompt the user to install those
Alternatively include subrepositories

**Comment by oseph** [comment]
I encountered this same issue (on macOS). It seems the URL for batik currently found in the `java/libraries/svg/build.xml` file is outdated. Apache now hosts batik (and a newer 1.14 version) over here: https://archive.apache.org/dist/xmlgraphics/batik/binaries/

I've updated the URL (and batik version to 1.14) in the build file to the above URL, and was able to build Processing4 successfully afterwards.

Will post a pull request shortly.
